packages/client/hmi-client/src/workflow/operator/tera-operator-title.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+	<h5 v-if="title">{{ title }}</h5>
```

Comment:
I wonder if a slot would be better, as this is the only thing.

Reply:
What would the slot have other than an `h5`? I feel like this doesn't have to be a component.